Assets/Scripts/API/BlocksFile.cs

```diff
@@ -584,8 +584,7 @@ private void ReadRmbFldHeader(BinaryReader reader, int block)
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown3 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown4 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].FactionId = reader.ReadUInt16();
-                blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Sector = reader.ReadInt16();
```

Comment:
Ok. The player/ship data stored in the save file seems to store these together as a single value, but we can probably work around that.